.eslintignore

```diff
@@ -1,19 +1,27 @@
-polyfills/_ArrayIterator
-polyfills/Array/of
-polyfills/Array/prototype/values
-polyfills/atob
-polyfills/AudioContext
-polyfills/fetch
-polyfills/Function/prototype/bind
-polyfills/HTMLPictureElement
-polyfills/IntersectionObserver
-polyfills/Intl
-polyfills/JSON
-polyfills/navigator/sendBeacon
-polyfills/Promise
-polyfills/setImmediate
-polyfills/URL
-polyfills/UserTiming
-polyfills/WeakSet
-polyfills/~html5-elements
+# We ignore the polyfill.js files for third-party polyfills
+# because we do not control their implementation.
+
+# We do not ignore the config.json, detect.js or tests.js
+# because we do control their implementation.
+
+polyfills/_ArrayIterator/polyfill.js
```

Comment:
can't you do `polyfills/**/**/polyfill.js` or `polyfills/**/polyfill.js` or something like that?